From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Tue, 9 Jan 2018 11:02:24 +0100
Subject: locking: add types.h
Git-repo: git://git.kernel.org/pub/scm/linux/kernel/git/rt/linux-rt-devel.git
Git-commit: 1289b06974d64f244a26455fab699c6a1332f4bc
Patch-mainline: Queued in subsystem maintainer repository
References: SLE Realtime Extension

During the stable update the arm architecture did not compile anymore
due to missing definition of u16/32.

Cc: stable-rt@vger.kernel.org
Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Signed-off-by: Mike Galbraith <mgalbraith@suse.de>
---
 include/linux/spinlock_types_raw.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/spinlock_types_raw.h
+++ b/include/linux/spinlock_types_raw.h
@@ -1,6 +1,8 @@
 #ifndef __LINUX_SPINLOCK_TYPES_RAW_H
 #define __LINUX_SPINLOCK_TYPES_RAW_H
 
+#include <linux/types.h>
+
 #if defined(CONFIG_SMP)
 # include <asm/spinlock_types.h>
 #else
